librosa/display.py

```diff
@@ -395,7 +412,7 @@ def __init__(
             n_bins, fmin=fmin, intervals=intervals, bins_per_octave=bins_per_octave
         )
 
-    def __call__(self, x, pos=None):
+    def __call__(self, x: float, pos: Optional[int] = None) -> str:
```

Comment:
do we know that pos should be int?

Reply:
yes, it's the index of the tick (x is the value)